src/app/products_view/product_availability_details/product_availability_details.ng.html

```diff
@@ -0,0 +1,21 @@
+<div class="container">
+  <mat-form-field appearance="outline" *ngIf="productQuantityInfo.length > 0">
```

Comment:
Please sort attributes according to the https://github.com/ooow/ap-lab#attributes